sdks/python/apache_beam/transforms/core.py

```diff
@@ -1906,7 +1920,7 @@ def __init__(self, value):
       value: An object of values for the PCollection
     """
     super(Create, self).__init__()
-    if isinstance(value, string_types):
+    if isinstance(value, (unicode, str, bytes)):
```

Comment:
See https://github.com/apache/beam/pull/5729#discussion_r199660012. Bytes in Python3 also shouldn't be allowed since we don't want to support creation of a PCollection of single bytes.